main.c

```diff
@@ -0,0 +1,27 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+typedef struct 
+{
+  char name[28];	

```

Comment:
Avoid using magic numbers. Should be used definition for this value

Reply:
done